components/dashboard/src/components/RepositoryFinder.tsx

```diff
@@ -87,15 +92,28 @@ export default function RepositoryFinder({
         // searchString ignore here as list is already pre-filtered against it
         // w/ mirrored state via useUnifiedRepositorySearch
         (searchString: string) => {
-            return repos.map((repo) => {
+            const result = repos.map((repo) => {
                 return {
                     id: repo.projectId || repo.url,
                     element: <SuggestedRepositoryOption repo={repo} />,
                     isSelectable: true,
                 } as ComboboxElement;
             });
+            if (hasMore) {
```

Comment:
I wonder if we should check `isLoading` and `isSearching` here as well, and not add the item if either of those are true? I think this would result in us not showing the entry when a search is happening, which seems nice.

Reply:
I did that just now, and the result i that the label flickers when I change the search string. I don't think this is helpful